### DISCUSSION TITLE
Relay endpoint is missing actions

### DISCUSSION THREAD

**Discussion by charklewis** [hasura/graphql-engine#7371] [Question]
I am using relay for a front end app and noticed it is missing the action handlers, how can I add them?

**Comment by coco98** [comment]
@charklewis We don't support some features on Relay yet, including Actions. https://hasura.io/docs/latest/graphql/core/databases/postgres/schema/relay-schema.html#exporting-the-relay-schema

Underneath, the biggest challenge here is implementing the node spec / GUID when multiple sources are involved.

**Reply by charklewis** [reply]
Ah that is shame, has it been put on the roadmap for an upcoming release?

**Reply by zth** [reply]
@coco98 hey Tanmai! 👋  / Gabriel

Jumping into the discussion here - would it be possible to have an initial implementation of actions for the Relay schema that simply did not implement the Node interface, or are the problems "before" that step? A rudimentary, initial version. I think it might be possible to get around the multi-sources-Node-interface issue with a few clever tricks for a lot of use cases.

Happy to throw ideas around on this topic! I'm (obviously) very interested in seeing Hasura be a viable alternative for Relay 😄
